tmt/cli.py

```diff
@@ -1844,15 +1844,15 @@ def clean_runs(
         context: Context,
         workdir_root: str,
         last: bool,
-        id_: Optional[str],
+        id_: tuple[str],
         keep: Optional[int],
         **kwargs: Any) -> None:
     """
     Clean workdirs of past runs.
 
     Remove all runs in '/var/tmp/tmt' by default.
     """
-    defined = [last is True, id_ is not None, keep is not None]
+    defined = [last is True, len(id_) > 0, keep is not None]
```

Comment:
`bool(id_)` should be more Pythonic "emptiness" of a sequence, similar to `if id_`.